src/unity/python/turicreate/data_structures/sframe.py

```diff
@@ -4973,7 +4976,31 @@ def unpack(self, column_name, column_name_prefix=None, column_types=None,
         | 3  |    3.0    |    4.0    |    5.0    |
         +----+-----------+-----------+-----------+
         [3 rows x 4 columns]
+        
+        >>> sf = turicreate.SFrame([{'a':1,'b':2,'c':3},{'a':4,'b':5,'c':6}])
+        >>> sf.unpack()
+        +---+---+---+
+        | a | b | c |
+        +---+---+---+
+        | 1 | 2 | 3 |
+        | 4 | 5 | 6 |
+        +---+---+---+
+        [2 rows x 3 columns]
+
+
+
         """
+        if column_name==None:
+            if len(self.column_names())==1:
```

Comment:
I would just do: `if self.num_columns() == 1:`

I wonder if maybe we need a check, and a separate error message, for when the number of columns is zero.

Reply:
Cool. I will make the changes. I think it returns a KeyError when there are zero columns. This was already in code though.

Reply:
I get it now the case of zero columns.